packages/react-sdk/src/core/components/ParticipantView/ParticipantView.tsx

```diff
@@ -160,12 +160,13 @@ export const ParticipantView = forwardRef<HTMLDivElement, ParticipantViewProps>(
         )}
       >
         <ParticipantViewContext.Provider value={participantViewContextValue}>
-          <Audio
-            // mute the local participant, as we don't want to hear ourselves
-            muted={isLocalParticipant || muteAudio}
-            sinkId={localParticipant?.audioOutputDeviceId}
-            audioStream={audioStream}
-          />
+          {/* mute the local participant, as we don't want to hear ourselves */}
+          {(isLocalParticipant || muteAudio) && (
```

Comment:
Shouldn't this be `!isLocalParticipant && !muteAudio`?

Reply:
🤦‍♂️ You are right! I shouldn't be working... fixing right away. 👍